extensions/ql-vscode/src/config.ts

```diff
@@ -111,13 +112,16 @@ const DISTRIBUTION_CHANGE_SETTINGS = [
   PERSONAL_ACCESS_TOKEN_SETTING,
 ];
 
+export type CLIChannel = "released" | "nightly";
```

Comment:
Perhaps call it `stable` instead of `released`? Another alternative would be to just make `channel` optional and make it possible to be `undefined`.

Reply:
I'm happy to change to "stable". To be honest I think that's a better name. I think I went with "released" because that terminology had been used in the internal issue, but now I look again I can't really see it.

I'd prefer to keep this enum with two values rather than being potentially undefined, just so it's clearer what the non-nightly option is.